jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -429,7 +429,7 @@ private static void configureContainerizer(
 
     rawConfiguration.getToTags().forEach(containerizer::withAdditionalTag);
 
-    if (rawConfiguration.getUseOnlyProjectCache()) {
+    if (Boolean.getBoolean(PropertyNames.USE_ONLY_PROJECT_CACHE)) {
```

Comment:
Just saying, I see we call `setBaseImageLayersCache()` a few lines above by default. So that could be
```
    containerizer
        ...
        .setBaseImageLayersCache(
            getCheckedCacheDirectory(
                PropertyNames.BASE_IMAGE_CACHE,
                Boolean.getBoolean(PropertyNames.USE_ONLY_PROJECT_CACHE)
                    ? projectProperties.getDefaultCacheDirectory()
                    : Containerizer.DEFAULT_BASE_CACHE_DIRECTORY))
        .setApplicationLayersCache(
```
But don't mind if this doesn't look any better.